js/activity.js

```diff
@@ -1461,7 +1461,7 @@ function Activity() {
         }
 
         __heightBasedScroll();
-		
+
 		let closeAnyOpenMenusAndLabels = function () {
```

Comment:
Please check indentation

Reply:
Line `1464` actually was an indented empty line. My editor removed it on save.

`1465` looks different because is it indented with `tab`. I reworked the indentation in e395468c9a95bca55809b7de148e4e832d3c4c05 - replaced all `tabs` with `4 spaces`.